sxs/waveforms/format_handlers/rotating_paired_diff_multishuffle_bzip2.py

```diff
@@ -134,6 +138,13 @@ def save(
     if formats is None:
         formats = sxs_formats
 
+    if corotating_frame_tolerance is None:
+        corotating_frame_tolerance = L2norm_fractional_tolerance
+        if L2norm_fractional_tolerance is None or L2norm_fractional_tolerance <=0.0:
+            warning = f"corotating_frame_tolerance has not been specified, so it is defaulting to L2norm_fractional_tolerance.  However, L2norm_fractional_tolerance is {L2norm_fractional_tolerance} which does not make sense as a corotating_frame_tolerance.  You probably want to explicitly set corotating_frame_tolerance."
+            warnings.warn(warning)
+            
+
```

Comment:
The minimum sensible tolerance is 100*epsilon, so let's just fall back to that.

```suggestion
    if corotating_frame_tolerance is None:
        corotating_frame_tolerance = max(L2norm_fractional_tolerance, 100*np.finfo(float).eps)
```